### ISSUE TITLE
Feature request: grit browse

### ISSUE THREAD

**Issue by ezzatron** [jmalloc/grit#56]
```
NAME:
   grit browse - Open a browser window to the GitHub project associated with <slug>.

USAGE:
   grit browse <slug>
```

**Comment by jmalloc** [comment]
Blocked by #57

**Comment by jmalloc** [comment]
Partially addressed in v0.6.7 -- naively assumes all repos are on GitHub or a GHE installation.

**Comment by jmalloc** [comment]
I was thinking about changing the `open` (alias `browse`) command to navigate to the current branch/tag/hash in the code view, rather than always loading the default branch. Thoughts?

**Comment by ezzatron** [comment]
Seems useful to me. Perhaps a flag to open the default instead would also be good.

**Comment by jmalloc** [comment]
Just as a note, the official [Github CLI](https://github.com/cli/cli) also includes a similar feature in the `gh repo view -w` command, which is a bit wordy, but could be aliased.

I had been wondering whether this command has a place in `grit` going forward, though I think with the addition of the feature mentioned above (navigating to the current branch, etc) and the fact that grit is not actually github-specific it probably still deserves a place.

**Comment by ezzatron** [comment]
Another small thing that would be nice here. A way to directly open the "homepage" URL set on a GitHub repo. E.g. on a Terraform config repo I usually set this to link to the workspace on Terraform Cloud, and if you could to `grit open homepage` (or similar), you could get a lot of power out of a simple feature.

**Comment by ezzatron** [comment]
@jmalloc Could we make `grit open` work from sub-directories of a Git repo? So basically, mirror how Git behaves when you run it from a sub-directory.
